README.md

```diff
@@ -86,15 +99,24 @@ source ./.venv/bin/activate
 pip install -r requirements.txt
 ```
 
+## To improve speed you might want to use *uv* to install requirements.txt
```

Comment:
Improve speed in what? How does `uv` come in to play here?

Reply:
It installs requirements much faster than default pip does. But as I said, it's just a suggestion.